packages/client/hmi-client/src/workflow/tera-workflow.vue

```diff
@@ -356,116 +357,127 @@ const removeNode = (event) => {
 	workflowService.removeNode(wf.value, event);
 };
 
-const largeNode = { width: 420, height: 220 };
+const addOperatorToWorkflow: Function = (operator: any) => () => {
+	workflowService.addNode(wf.value, operator.operation, newNodePosition);
+	workflowDirty = true;
+};
+const addLargeOperatorToWorkflow: Function = (operator: any) => () => {
+	workflowService.addNode(wf.value, operator.operation, newNodePosition, {
+		size: { width: 420, height: 220 }
+	});
+	workflowDirty = true;
+};
```

Comment:
Would prefer to just have a single method that passes in optional size, rather than normal, large, and what other sizes that comes up.

```
addOperatorToWorkflow(operator, size?) 
```

Reply:
Also technically speaking `operator` is typed, the typing is available in services/workflow.ts though it may not be exported.

```
interface OperatorImport {
  name: string;
  operation: Operation;
  node: Component;
  drilldown: Component;
}

```

Reply:
Both are done